### PR TITLE
add chromium

### PR DESCRIPTION
**Pull request by oberon2007**
If the plan is to [collect browser settings here](https://github.com/manjaro/manjaro-firefox-settings/issues/2), this could be the version with settings for FF, Palemoon and Chromium already.
The chromium **Preferences** file was copied from a brand new profile with no page ever loaded. I just used the settings tab to configure the bookmarks and startpage.
Of course it does contain some other "dirty" information. I've played around with it a bit, but removing any of the rest seems to break it.
The **Bookmarks** file would also work on its own.
So it looks like if we don't want to ship the Preferences file we'd have to lose the pre-configured startup page until we maybe find a totally clean way similar to the FF distribution.ini
But I think just using it should not be a problem, either.

### CODE REVIEW

**Comment by Kirek** [comment]
https://www.chromium.org/administrators/configuring-other-preferences
master_preferences should be in /usr/lib/chromium/
We could put the Bookmarks file in there too

**Comment by oberon2007** [comment]
Brilliant! :laughing: You just need to know where to look for the stuff! :stuck_out_tongue_winking_eye: 
Shall I attempt at creating one?
It even looks like this will also work from Chrome then...

**Comment by oberon2007** [comment]
Ok. would you like to rename the repo, maybe? I can adjust the PKGBUILD then with the new stuff :)

File: chrome/master_preferences [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  },
+  "session" : {
+    "restore_on_startup" : 4,
+    "startup_urls" : [
+      "https://manjaro.github.io"
+    ]
+  },
+  "distribution": {
+    "import_bookmarks": false,
+    "import_bookmarks_from_file": "\/usr\/lib\/chromium\/bookmarks.html",
```
Comment (Kirek) [inline chrome/master_preferences]: Didn't try the config myself, but probably in linux it doesn't need to be escaped

Comment (oberon2007) [inline chrome/master_preferences]: In any case it works like that :)
Will copy the icon to Home and Startpage, too...

Comment (oberon2007) [inline chrome/master_preferences]: ah. no. forget what I just said about the icons. I meant I copied them to all three bookmarks ;)

Comment (oberon2007) [inline chrome/master_preferences]: ok. just verified. unescaped works, too :+1:

**Pull request merged**